modules/nodes-scheduler-taints-tolerations-about.adoc

```diff
@@ -216,9 +244,23 @@ These tolerations ensure that the default pod behavior is to remain bound for fi
 
 You can configure these tolerations as needed. For example, if you have an application with a lot of local state, you might want to keep the pods bound to node for a longer time in the event of network partition, allowing for the partition to recover and avoiding pod eviction.
 
-Daemon set pods are created with NoExecute tolerations for the following taints with no tolerationSeconds:
+DaemonSet pods are created with NoExecute tolerations for the following taints with no tolerationSeconds:
 
 * `node.kubernetes.io/unreachable`
 * `node.kubernetes.io/not-ready`
 
-This ensures that daemon set pods are never evicted due to these node conditions, even if the `DefaultTolerationSeconds` admission controller is disabled.
+This ensures that DaemonSet pods are never evicted due to these node conditions, even if the `DefaultTolerationSeconds` admission controller is disabled.
```

Comment:
What's _this_?

Reply:
> due to prep
> Do not use to mean “because of.” Use “due to” only in adjectival clauses. Although the use of “due to” as a synonym of “because of” has some acceptance in modern English usage, it is imprecise and, therefore, not suited to technical writing.